frappe/website/utils.py

```diff
@@ -605,3 +607,9 @@ def is_binary_file(path):
 	with open(path, "rb") as f:
 		content = f.read(1024)
 		return bool(content.translate(None, textchars))
+
+
+def check_if_webform_exists(route):
+	return frappe.db.exists("Web Form", {"name": route.strip("/")}) or frappe.db.exists(
+		"Web Form", {"route": route.strip("/")}
+	)
```

Comment:
web form only renders from route:
https://github.com/frappe/frappe/blob/85e79537c88c5f39509b62a156953d5fb69f4149/frappe/website/router.py#L32

do we need the check for name?